.changeset/tiny-lions-watch.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-components': minor
+'@backstage/plugin-user-settings': minor
+---
+
+Added `signIn` and `signOut` analytic events to the core-components of signin and signout. used `userEntityRef` for determining the user
```

Comment:
Looks like `signin` and `signout` are not official words in the dictionary. 🙃 

Can you change these to Sign In and Sign Out respectively?

Also both these packages should be `patch` as they're both `0.x` packages. `minor` means breaking changes for these packages, of which they are not.